src/main/java/mate/academy/internetshop3/dao/BucketDao.java

```diff
@@ -13,5 +14,11 @@
 
     void delete(Long id);
 
-    void deleteItem(Item item);
+    Long getBucketId(Long userId);
```

Comment:
```suggestion
    Bucket getBucketByUserId(Long userId);
```
Краще витягувати об'єкт вцілому. Тоді ти зможеш з об'єкту отримати і bucketId і будь-які інші параметри. Це хороший спосіб для перевикористання твого коду